tests/Integration/Endpoints/AnalyticsPostsProxyEndpointTest.php

```diff
@@ -54,7 +52,7 @@ public function get_endpoint(): Base_API_Proxy {
 	 * @uses \Parsely\RemoteAPI\Remote_API_Base::__construct
 	 */
 	public function test_register_routes_by_default(): void {
-		parent::test_register_routes_by_default();
+		parent::run_test_register_routes_by_default();
```

Comment:
Renamed all related methods as such in order to reduce invalid complaints from PHPCS and messages that the methods needed a `covers` comment in the base class.